data/hai/hai reveal 1 intro.txt

```diff
@@ -664,22 +685,19 @@ mission "Hai Leaks Response 2C"
 				has "met remington"
 			action
 				log "People" "Samuel Remington" `Samuel Remington is the Syndicate's head of public relations. He is acting as the Syndicate's representative in helping to deal with the situation regarding the Hai.`
-			`	"Captain <last>, is it? I don't believe we've met before," he says while shaking your hand, perhaps a little too firmly. "I'm Samuel Remington, head of the Syndicate's PR department. You may have not heard of me before but I've sure you've heard of my work. I..." He stops himself. "You know what, nevermind. We're not here to make small talk. We have bigger things to worry about. I'm looking forward to working alongside you to handle this emerging crisis.`
+			`	"Captain <last>, is it? I don't believe we've met before," he says while shaking your hand, perhaps a little too firmly. "I'm Samuel Remington, head of the Syndicate's PR department."  He gestures to the woman behind him and says, "Let me introduce you to Terry."`
```

Comment:
```suggestion
			`	"Captain <last>, is it? I don't believe we've met before," he says while shaking your hand, perhaps a little too firmly. "I'm Samuel Remington, head of the Syndicate's PR department. I'm sure you've heard of my work. I..." He stops suddenly. "You know what, nevermind. We're not here to make small talk, there's more important matters to discuss. I look forward to working alongside you in handling this emerging crisis.`
```
Would this be better? If it were to be kept in, the repetition needs to be removed anyway, unless that's there because he seems to be talking so quickly.

Reply:
You are correct.